pom.xml

```diff
@@ -623,14 +623,13 @@
           <configuration>
             <excludes combine.children="append">
               <!-- Files that don't require a license header -->
-              <exclude>**/README.md</exclude>
               <exclude>**/LICENSE</exclude>
               <exclude>**/NOTICE</exclude>
-              <exclude>CONTRIBUTING.md</exclude>
               <exclude>.travis.yml</exclude>
 
               <exclude>**/target/**</exclude>
 
+              <exclude>**/*.md</exclude>

```

Comment:
If we are to exclude all `.md` files from rat plugin then we could also remove the exclusions from **documentation** module pom.